CHANGELOG.md

```diff
@@ -1,7 +1,21 @@
-Changelog
-=========
+# Changelog
 
-2.x
----
+## 2.0 [TBA]
+### Added
+ * Applied `declare(strict_types=1)` everywhere
```

Comment:
I think that it should be to the bottom of the list, this item is not a priority for end users.